src/tab/header/item/TabHeaderItem.tsx

```diff
@@ -1,17 +1,19 @@
 import React from "react";
+import classNames from "classnames";
 
 import {TabItem} from "../../Tab";
 import Button from "../../../button/Button";
 
 type TabHeaderItemProps = Omit<TabItem, "id"> & {
   index: number;
   onClick: (index: number) => void;
+  isActive: boolean;
 };
 
-function TabHeaderItem({children, icon, index, onClick}: TabHeaderItemProps) {
+function TabHeaderItem({children, icon, index, onClick, isActive}: TabHeaderItemProps) {
   return (
     <Button
-      customClassName={"tab-header-item"}
+      customClassName={classNames("tab-header-item", {active: isActive})}
```

Comment:
Shouldn't you wrap `active` with quotation marks? Like: `{"active": isActive}`

Reply:
I think our prettier rules automatically removes `"`s from keys if they don't include special characters (such as `-`)

Reply:
Agreed with other comments but let's use BEM convention here
```suggestion
      customClassName={classNames("tab-header-item", {"tab-header-item--is-active": isActive})}
```

Reply:
Done 👍